package.json

```diff
@@ -21,7 +27,8 @@
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-helmet": "^5.2.1"
+    "react-helmet": "^5.2.1",
+    "reactstrap": "^8.0.1"
```

Comment:
Similarly, any reason to use this over [`react-bootstrap`](https://github.com/react-bootstrap/react-bootstrap)? The latter seems to be used by a lot more people, and has a lot more stars and forks on GitHub, not to mention more and more recent commits. `reactstrap` has more open issues and PRs, but it seems that that might be due to less activity from the team and not more 😅

Reply:
I too made that suggestion but @tiagofragoso recommended `reactstrap` over the former, and we agreed.

Reply:
Fair point, mostly curious